app/models/concerns/orchestration/ssh_provision.rb

```diff
@@ -99,6 +99,6 @@ def validate_ssh_provisioning
 
   def provision_host
     # usually cloud compute resources provide IPs but virtualization do not
-    provision_interface.ip || provision_interface.fqdn
+    provision_interface.ip || provision_interface.ip6 || provision_interface.fqdn
```

Comment:
Like others: should we prefer IPv6 over IPv4? Users will appreciate it if we're consistent and I'd lean to preferring IPv4.

Reply:
So in current implementation we do implement IPv4, this way it's less surprising.

I tend to go with less surprises too

Reply:
DNS lookups typically also prefer IPv6 (which AFAIK is the non-IP option we already implement). For admins it's more consistent with how the world does it. If needed we can add a release note

Reply:
So if I understand you correctly, the current implementation is OK? IPv4, then IPv6 and then FQDN.

Reply:
No, IPv6 first and then IPv4 is what I'd prefer. We should start treating IPv4 as legacy just like the rest of the industry